.github/workflows/rust-av.yml

```diff
@@ -1,101 +1,109 @@
 name: rust-av
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '**.rs'
+      - '**.sh'
+      - '**.yml'
+      - '**.toml'
+      - '!**.md'
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
[The default is](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) `[opened, synchronize, reopened]`. I think we can just use the default.